### DISCUSSION TITLE
Display toast over React Navigation Modal

### DISCUSSION THREAD

**Discussion by hanselabreu** [arnnis/react-native-fast-toast#85] [Q&A] [answered]
Is there a known way of displaying a toast over a [React Navigation Modal](https://reactnavigation.org/docs/modal/)? Currently the toast is shown behind the modal.

**Comment by arnnis** [comment, marked as answer]
As explained [here](https://github.com/arnnis/react-native-toast-notifications#--how-to-show-toast-inside-a-modal) since Modal is a native view it goes above anything else, only workaround is to have a `<Toast />` component inside the modal itself.

**Reply by hanselabreu** [reply]
Thank you so much. Missed that on the documentation.

**Comment by Jamal-ReachFirst** [comment]
Then what's the benefit of using useToast()

**Reply by carrie503** [reply]
I have written another workaroud so we can use `useToast`

See here:
https://github.com/arnnis/react-native-toast-notifications/issues/50#issuecomment-1637708341
